_unittest/test_01_Design.py

```diff
@@ -94,8 +94,8 @@ def test_08_objects(self):
         print(self.aedtapp.variable_manager)
         print(self.aedtapp.materials)
 
-    def test_09_add_workbench_link(self):
-        assert self.aedtapp.modeler.add_workbench_link(["inner"], "25")
+    def test_09_set_objects_deformation(self):
+        assert self.aedtapp.modeler.set_objects_deformation(["inner"])
```

Comment:
I'd like for us to validate more than just the method ran.  Perhaps if we also has (and this code probably isn't right):
```
        assert self.aedtapp.modeler.get_objects_deformation(["inner"]) == 25
```

Reply:
I agree with your suggestion, but this method just set a flag option and AFAIK there is no API call to check its status.